src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardView/HomeStatisticsCardView.swift

```diff
@@ -111,6 +115,11 @@ class HomeStatisticsCardView: UIView {
 					primaryAccessibilityLabel?.append(primaryValueLabel.text ?? "")
 				}
 
+				if viewModel?.primarySubtitle != nil, let primarySubtitleLabel = self.primarySubtitleLabel {
+					primaryAccessibilityLabel?.append(" ")
```

Comment:
Should this also be appended if `primarySubtitleLabel.text` is nil?

Reply:
Thanks for that hint. It doesn't actually make much sense to go through `primarySubtitleLabel.text` if I have the value itself readily available. Simplified it now :)